pkgs/by-name/ka/kafka-cmak/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  jdk,
+  gawk,
+  makeWrapper,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "CMAK";
+  version = "3.0.0.6";
+
+  src = fetchurl {
+    url = "https://github.com/yahoo/CMAK/releases/latest/download/cmak-${finalAttrs.version}.zip";
+    hash = "sha256-PolFcvFagDTR1CUee2gVXdxZzjioobiiHWUUCUbCGe8=";
+  };
+
+  buildInputs = [
+    gawk
+    unzip
+    makeWrapper
+    jdk
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv ./* $out
+    wrapProgram $out/bin/cmak \
```

Comment:
Since this is available on Mac, I would prefer to use `makeBinaryWrapper` since if offers a better compatibility with this architecture.

Reply:
Ohh right, I probably should start to default to using `makeBinaryWrapper` anyways >_>